synapse/config/push.py

```diff
@@ -68,4 +71,14 @@ def generate_config_section(self, config_dir_path, server_name, **kwargs):
           # include the event ID and room ID in push notification payloads.
           #
           #include_content: false
+
+          # When a push notification is received, a total unread message count is also sent.
+          # This number can either be calculated as the total number of unread messages
+          # for the user, or the total number of *rooms* the user has unread messages in.
```

Comment:
Would need reflowing, but maybe saying an "unread count" in the first sentence is a bit clearer?

```suggestion
          # When a push notification is received, an unread count is also sent.
          # This number can either be calculated as the total number of unread messages
          # for the user, or the total number of *rooms* the user has unread messages in.
```

I also find the use of "total" distracting in here, what's the alternative?

Reply:
Indeed, the paragraphs still carry well after removing "total" from each of them :+1: